dart/lib/src/protocol/sentry_event.dart

```diff
@@ -262,12 +263,15 @@ class SentryEvent {
       json['exception'] = {
         'values': [exception.toJson()].toList(growable: false)
       };
-    } else if (stackTrace is SentryStackTrace) {
+    }
+
+    if (stackTrace != null) {
       json['threads'] = {
         'values': [
           {
-            'id': 0,
-            'stacktrace': (stackTrace as SentryStackTrace).toJson(),
+            'id': Isolate.current.debugName,
```

Comment:
`debugName` is a String, `id` is a name, why have you changed it?
threads have a `name` field

Reply:
✅